src/Host.cpp

```diff
@@ -639,8 +635,8 @@ void Host::reloadModule(const QString& syncModuleName, const QString& syncingFro
     //Wait till profile is finished saving
     if (syncingFromHost.isEmpty() && currentlySavingProfile()) {
         //create a dummy object to singleshot connect (disconnect/delete after execution)
-        QObject *obj = new QObject(this);
-        connect(this, &Host::profileSaveFinished, obj, [=](){
+        QObject* obj = new QObject(this);
+        connect(this, &Host::profileSaveFinished, obj, [=]() {
             reloadModule(syncModuleName);
             obj->deleteLater();
         });
```

Comment:
I've since added a thing to do singleshot connections with, https://github.com/Mudlet/Mudlet/blob/development/src/dlgProfilePreferences.cpp#L4030 - want to switch to that?